### PR TITLE
🔀 Merge main to v1

### PR DESCRIPTION
**Pull request by ashwinvaidya17**
## 📝 Description

Provide a clear summary of the changes and the issue that has been addressed.

🛠️ Fixes # (issue number)

## ✨ Changes

Select what type of change your PR is:

- [ ] 🐞 Bug fix (non-breaking change which fixes an issue)
- [ ] 🔨 Refactor (non-breaking change which refactors the code base)
- [ ] 🚀 New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📚 Documentation update

## ✅ Checklist

Before you submit your pull request, please make sure you have completed the following steps:

- [ ] 📋 I have summarized my changes in the [CHANGELOG](https://github.com/openvinotoolkit/anomalib/blob/main/CHANGELOG.md) and followed the guidelines for my type of change (skip for minor changes, documentation updates, and test enhancements).
- [ ] 📚 I have made the necessary updates to the documentation (if applicable).
- [ ] 🧪 I have written tests that support my changes and prove that my fix is effective or my feature works (if applicable).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,8 @@
 [![python](https://img.shields.io/badge/python-3.7%2B-green)]()
 [![pytorch](https://img.shields.io/badge/pytorch-1.8.1%2B-orange)]()
 [![openvino](https://img.shields.io/badge/openvino-2022.3.0-purple)]()
-<br/>
+
+[![comet](https://custom-icon-badges.herokuapp.com/badge/comet__ml-3.31.7-orange?logo=logo_comet_ml)](https://www.comet.com/site/products/ml-experiment-tracking/?utm_source=anomalib&utm_medium=referral)
```
Comment (samet-akcay) [inline README.md:18]: should we include this badge in v1?

File: src/anomalib/models/image/dsr/lightning_model.py [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+    url="https://github.com/openvinotoolkit/anomalib/releases/download/dsr_pretrained_weights/dsr_vq_model_pretrained.zip",
+    checksum="927f6b40841a7c885d12217c922b2bba",
+)
+
+
+class Dsr(AnomalyModule):
+    """DSR: A Dual Subspace Re-Projection Network for Surface Anomaly Detection.
+
+    Args:
+        latent_anomaly_strength (float, optional): Strength of the generated anomalies in the latent space.
```
Comment (samet-akcay) [inline src/anomalib/models/image/dsr/lightning_model.py:40]: would it be possible to update this docstring

File: src/anomalib/models/image/dsr/lightning_model.py [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+        if not (pretrained_models_dir / "vq_model_pretrained_128_4096.pckl").is_file():
+            download_and_extract(pretrained_models_dir, WEIGHTS_DOWNLOAD_INFO)
+        return pretrained_models_dir / "vq_model_pretrained_128_4096.pckl"
+
+    def configure_optimizers(
+        self,
+    ) -> tuple[
+        dict[str, torch.optim.Optimizer | torch.optim.lr_scheduler.LRScheduler],
+        dict[str, torch.optim.Optimizer],
+    ]:
```
Comment (samet-akcay) [inline src/anomalib/models/image/dsr/lightning_model.py:69]: is this annotation different than lightning's output annotation?

File: src/anomalib/models/image/uflow/anomaly_map.py [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+        normal distribution. The null-hypothesis is that under normality assumptions, all candidate pixels are uniformly
+        distributed. Then, the detection is based on the concentration of candidate pixels.
+
+        Args:
+            z: List of latent variables from the UFlow model. Each element is a tensor of shape
+            (N, Cl, Hl, Wl), where N is the batch size, Cl is the number of channels, and Hl and Wl are the height and
+            width of the latent variables, respectively, for each scale l.
+            window_size (int): Window size for the binomial test.
+            binomial_probability_thr: Probability threshold for the binomial test.
+            high_precision: Whether to use high precision for the binomial test.
```
Comment (samet-akcay) [inline src/anomalib/models/image/uflow/anomaly_map.py:85]: is it possible to add default values here?

**Review by samet-akcay — COMMENTED** [review]
thanks just minor comments

File: README.md (hunk 1 above)
Comment (ashwinvaidya17) [reply README.md:18]: I don't know. We still support comet.

**Pull request merged**